heat_pump_adoption_modelling/pipeline/MCS/load_mcs.py

```diff
@@ -0,0 +1,137 @@
+                "Total Installed Capacity": "capacity",
+                "Green Deal Installation?": "green_deal",
+                "Products": "products",
+                "Flow temp/SCOP ": "flow_scop",
+                "Technology Type": "tech_type",
+                " Installation Type": "installation_type",
+                "Installation New at Commissioning Date?": "new",
+                "Renewable System Design": "design",
+                "Annual Space Heating Demand": "heat_demand",
+                "Annual Water Heating Demand": "water_demand",
+                "Annual Space Heating Supplied": "heat_supplied",
+                "Annual Water Heating Supplied": "water_supplied",
+                "Installation Requires Metering?": "metering",
+                "RHI Metering Status": "rhi_status",
+                "RHI Metering Not Ready Reason": "rhi_not_ready",
+                "Number of MCS Certificates": "n_certificates",
+                # "RHI?": "rhi",
+                "Alternative Heating System Type": "alt_type",
+                "Alternative Heating System Fuel Type": "alt_fuel",
+                "Overall Cost": "cost",
+            }
+        )
+        .convert_dtypes()
+        .drop_duplicates()
+    )
+
+    # Make RHI field values easier to use
+    # Commented out as RHI field has disappeared from the most recent MCS data
+    # hps["rhi"] = hps["rhi"].replace(
+    #     {
+    #         "RHI Installation ": True,
+    #         "Not Domestic RHI installation ": False,
+    #         "Unspecified": np.nan,
+    #     }
+    # )
+
+    # Filter to domestic installations
+    dhps = (
+        hps[hps["installation_type"].isin(["Domestic", "Domestic "])]
+        .drop(columns="installation_type")
+        .reset_index(drop=True)
+    )
+
+    most_recent_indices = dhps.groupby(["address_1", "address_2", "address_3"])[
+        "version"
+    ].idxmax()
+
+    dhps = dhps.iloc[most_recent_indices]
+
+    # Extract information from product column
+    regex_dict = {
+        "product_name": "Product Name: ([^\|]+)",
+        "manufacturer": "License Holder: ([^\|]+)",
+        "flow_temp": "Flow Temp: ([^\|]+)",
+        "scop": "SCOP: ([^\)]+)",
+    }
+
+    for key, value in regex_dict.items():
+        dhps[key] = [
+            re.search(value, product).group(1).strip() for product in dhps.products
+        ]
+
+    # Replace unreasonable cost and capacity values with NA
+    dhps["cost"] = dhps["cost"].mask((dhps["cost"] == 0) | (dhps["cost"] > max_cost))
+    dhps["capacity"] = dhps["capacity"].mask(dhps["cost"] > max_capacity)
+
+    dhps = dhps.reset_index(drop=True)
+
+    return dhps
+
+
+def load_epcs():
+    """Loads relevant columns of EPC records.
+
+    Return
+    ----------
+    epcs: pandas.Dataframe
+        EPC records, columns specified in config.
+    """
+    epcs = pd.read_csv(
+        PROJECT_DIR / epc_path,
+        usecols=epc_address_fields + epc_characteristic_fields,
+    )
```

Comment:
In the future (after I finally merged utils_core to dev), you won't need that anymore:

```
from heat_pump_adoption_modelling.getters import epc_data

fields_of_interest = epc_address_fields + epc_characteristic_fields
 epcs = epc_data.load_preprocessed_epc_data(
    version="preprocessed_dedupl", usecols=fields_of_interest)
```

Reply:
Thanks - changed!